### DISCUSSION TITLE
Paid for, no proxy but still says "Not yet activated"

### DISCUSSION THREAD

**Discussion by nwthync** [community/community#36490] [Copilot Conversations]
Hi All, I have a subscription, window s10 machine, vs code with no proxy but I am still getting "Not yet activated"
Is there a way to force it to try activation again?

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by nwthync** [reply]
Thankyou, but where is the signout button in VS code?

**Reply by davecheney** [reply]
Bottom left, look for the silhouette 👤

**Comment by ossanna16** [comment]
Hey there @nwthync and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by rehypothecation** [comment]
Any solution to this? Nobody else seems to be having this problem.

**Reply by itrepablik** [reply]
I'm also facing this one right now, suddenly it's gone in VS code even though I follow the guidance on sign-in/out again, even disabling the extension but still failed. 😳

**Reply by itrepablik** [reply]
hi, after I tried many attempts try to fix this issue at my end when I keep `signing in/out` from `vs code`, and try `disabling, installing, and installing again back the copilot extension` it seems that the prompt to authenticate the account again is not poping-up when I open the project that I'm working on.  But, instead, when I close the project that I'm working on and open a `fresh vs code`, it shows up the pop-up authentication to sign in to Github for the copilot. Then, it's working back again.

**Comment by ouweiya** [comment]
There was a recent update about today which appears to be a corrupted or broken version of Github Copilot. You need to install an older version of Github Copilot by going into the extension page thens earching for 'Github Copilot' and clicking the arrow button next to 'Uninstall'. From there, click the tab which says 'Install Another Version' and install v1.63.7601 (x days ago).

You will be prompted to reload VS Code and Github Copilot should begin working again.

https://github.com/community/community/discussions/41962#discussioncomment-4435896

**Comment by simon-contreras-deel** [comment]
I have this problem. I tried `signing in/out` and `enabling/disabling`.

I am using Mac 13.2 and the copilot version is `v1.82.15`

**Comment by pkws100** [comment]
I have the same problem on version 1.82.36

**Comment by simon-contreras-deel** [comment]
Still happening now in version `v1.83.41`.

In my case, I see that the problem is happening only when working with the Microsoft extension [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
